Main/Src/classes/CQ_TASK_Test_Complaint_UpdateFlowTest.cls

```diff
@@ -0,0 +1,121 @@
+        PermissionSetGroup psg = [select Id, Status from PermissionSetGroup where DeveloperName='CQ_Base_User_PSG'];
+        
+        //Arrange : force calculation of the PSG if it is not already Updated
+        if (psg.Status != 'Updated') {
+            Test.calculatePermissionSetGroup(psg.Id);
+        }
+        
+        //Arrange : assign PSG to standard user (this fails if PSG is Outdated)
+        insert new PermissionSetAssignment(PermissionSetGroupId = psg.Id, AssigneeId = standardUser.Id);
+
+        System.runAs(standardUser) {        
+            //Arrange : Create Account Record for Complaint. 
+            Account accountRecord = new Account(Name='Test Account');
+            insert accountRecord;
+
+            //Arrange : Create Contact Record for Complaint.
+            Contact contactRecord = new Contact(LastName='Test',FirstName='Contact',AccountId=accountRecord.Id);
+            insert contactRecord;
+
+            //Arrange : Create Investigation Record with Complaint Investigation RecordType for Complaint.
+            compliancequest__SQX_Investigation__c inestigationRecord = new compliancequest__SQX_Investigation__c();
+            inestigationRecord.recordTypeID = Schema.SObjectType.compliancequest__SQX_Investigation__c.getRecordTypeInfosByDeveloperName().get('Complaint_Investigation').getRecordTypeId();
+            inestigationRecord.compliancequest__Investigation_Summary__c = 'Test Summary';
+            inestigationRecord.compliancequest__Conclusion__c = 'Confirmed';
+            insert inestigationRecord;
+
+            //Arrange : Create Compaint Record.
+            compliancequest__SQX_Complaint__c complaintRecord = new compliancequest__SQX_Complaint__c();
+            complaintRecord.compliancequest__Complaint_Title__c = 'Test Complaint';
+            complaintRecord.compliancequest__SQX_Account__c = accountRecord.Id;
+            complaintRecord.compliancequest__SQX_External_Contact__c = contactRecord.Id;
+            complaintRecord.compliancequest__Country_of_Origin__c = 'IN';        
+            insert complaintRecord;        
+
+            //Arrange : Create Linked Investigation record to Link Complaint and Investigation Records.
+            compliancequest__SQX_Linked_Investigation__c linkedInvestigationRecord = new compliancequest__SQX_Linked_Investigation__c();
+            linkedInvestigationRecord.compliancequest__SQX_Complaint__c = complaintRecord.Id;
+            linkedInvestigationRecord.compliancequest__SQX_Investigation__c = inestigationRecord.Id;
+            insert linkedInvestigationRecord;
+        }
+    }
+
+    /**
+     * GIVEN: A Investigation Record
+     * WHEN : Status = Completed
+     * THEN : Update Investigation Summary and Result of Investigtion fields on Complaint 
+     *          from Investigation Summary and Conclusion fields.
+     * 
+     * Flow : CQ_TASK_Investigation_After_Update_RT_Flow
+     *        CQ_Complaint_OnInvestigationComplete_SubFlow
```

Comment:
update flow name

Reply:
Updated floe name to CQ_TASK_Complaint_OnInvestigationComplete_SubFlow